swift/storage.py

```diff
@@ -262,13 +262,15 @@ def _save(self, name, content, headers=None):
             content.seek(0)
         else:
             content_type = mimetypes.guess_type(name)[0]
+        content_length = os.fstat(content.fileno()).st_size
```

Comment:
Assuming we always get a Django `File` object or one if their subclasses we should probably use `File.size` to get the byte size of the object.